README.md

```diff
@@ -18,15 +19,34 @@ in the Fedora Commons Repository at the dataset level. This command extracts the
 Fedora Commons Repository and sends it to the EASY SOLR Search Index to add or update the record for the specified
 dataset(s)
 
+
 ARGUMENTS
 ---------
 
-* ``-u``, ``--user`` -- Fedora user to connect with
-* ``-p``, ``--password`` -- password of the Fedora user
-* ``-f``, ``--fcrepo-server`` -- URL of the Fedora Commons Repository server
-* ``-s``, ``--solr-server-request-handler`` -- URL of the EASY SOLR Search Index server request handler
-* ``-x``, ``--output-xml-dir`` -- directory to write the documents to be sent to SOLR to
-* ``<easy-datataset-pid>...`` -- one or more dataset PIDs 
+      -a, --apply-updates               If omitted: only generate document(s), do not
```

Comment:
I disagree with this change. For tasks that run the risk to mess up the actual data or metadata that may be a good thing. This command however just updates the search index. Would be annoying to have to specify everytime that you really mean to do it.